cloud/migrate-to-cloud/hypershift.md

```diff
@@ -258,19 +258,26 @@ include_schemas:
 - '"FooBar"'
 ```
 
-Note: as with `pg_dump`, if there is a dependency from one schema to an object
-in another schema which is not included, the dependency will be unfulfilled,
-resulting in failure.
 
-Mutually exclusive with: `exclude_schemas`
+<highlight type="note">
+It is possible for objects in one schema to depend on objects in another schema
+for example, a table in the `public` schema depending on a type in the `other`
+schema. Hypershift will not warn about missing dependencies on objects in
+schemas which are not included, instead the migration will fail.
+
+To avoid this, ensure that you do not exclude schemas containing objects on
+which objects in other schemas depend.
```

Comment:
```suggestion
To avoid this, do not exclude schemas with objects that are depended on 
by objects in other schemas.
```

Reply:
Hope I didn't change the meaning here, just trying to make the sentence less complicated 😅